packages/integrations/vercel/test/edge-middleware.test.js

```diff
@@ -38,9 +40,9 @@ describe('Vercel edge middleware', () => {
 			// this is abysmal...
 			'../.vercel/output/functions/render.func/www/withastro/astro/packages/integrations/vercel/test/fixtures/middleware-with-edge-file/dist/middleware.mjs'
 		);
-		expect(contents.includes('title:')).to.be.true;
-		chaiJestSnapshot.setTestName('Middleware with handler file');
-		expect(contents).to.matchSnapshot(true);
+		// assert.equal(contents.includes('title:')).to.be.true;
+		// chaiJestSnapshot.setTestName('Middleware with handler file');
+		// assert.equal(contents).to.matchSnapshot(true);
```

Comment:
What's up with these?

Reply:
They were snapshot tests, but they didn't go very well; in fact, they were skipped. We could delete them, but best to do it in a later PR